### PR TITLE
Clean up of the sbt build file.

### PR DESCRIPTION
**Pull request by gpoirier**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -1,20 +1,16 @@
...
+import scalariform.formatter.preferences._
 
 enablePlugins(GitVersioning)
 git.useGitDescribe := true
 
-lazy val commonSettings = Seq(
+lazy val commonSettings = Jvm.required(Jvm.V7) ++ Seq(
+  organization       := "com.mediative",
+  scalaVersion       := "2.10.5",
+  crossScalaVersions := Seq("2.10.5", "2.11.7"),
```
Comment (gpoirier) [inline build.sbt]: Initially I had both 2.10.4 and 2.10.5 here, but I figured with our dependencies, the version get upgraded to 2.10.5 anyway, so I'm not sure the 2.10.4 build was doing any good and just slow things don't when you want locally to make sure tests work with `+ test`. Travis is still configured with all 3 version though.

**Comment by jonas** [comment]
Any reason to test with openjdk7 instead of oraclejdk7?

Otherwise nice cleanup: +1

**Comment by gpoirier** [comment]
I wanted to have `openjdk7` and `openjdk8`, because if I works on open JDK I don't really see why it wouldn't on Oracle JDK. But Travis doesn't support `openjdk8`, so I had to change it to `oraclejdk8`. So I figured this way it get tested on both OpenJDK and Oracle JDK even if not all 4 combinations. If you feel like other versions are more appropriate of want to add `oraclejdk7` in the mix, feel free to make the change.

**Comment by gpoirier** [comment]
Although I guess the limitations of openjdk only affect graphics (I think) and would be unlikely to cause problem in a library like this one.

**Pull request merged**